charts/piraeus/values.yaml

```diff
@@ -35,3 +35,4 @@ operator:
     kernelModImage: "quay.io/piraeusdatastore/drbd9-bionic:v9.0.22"
     storagePools: null
     sslSecret: ""
+    automaticStorageSetupType: None
```

Comment:
I think we should switch this on by default to reduce the number of steps required for someone trying out the operator to get to a working system. Defaut `LVMTHIN`.